src/EODList.js

```diff
@@ -7,13 +7,21 @@ const sleepyRAPath = path.join(__dirname, '../config-files/sleepyRAs.txt');
 var sleepyRANames = fs.readFileSync(sleepyRAPath).toString().split('\n');
 const EODPath = path.join(__dirname, "../config-files/eods.json");
 var EODs = JSON.parse(fs.readFileSync(EODPath));
-var EODContent = fs.readFileSync(EODReminderPath);
+const EODContent = fs.readFileSync(EODReminderPath);
 var EODJSON = JSON.parse(EODContent);
 const { execSync } = require('child_process');
 
+// Check if user is in channelID
+const checkUserChannelID = (name) => {
+    const channelIDs = require(path.join(__dirname, '../config-files/channelID.json'));
```

Comment:
Do not require inside functions. Place requires at the top of the file.

Reply:
The point of requiring inside the function is so it gets the most recently updated channelID.json when the function is called.

Reply:
Your config file is changing at runtime? Okay, but one drawback is that `require` is synchronous. That's alright for now, but it may be worth opening an issue at some point to rewrite all the synchronous operations that exist in the codebase (reads, writes, and executes are all sync).